app/controllers/api/docs/openapi_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module API
+  module Docs
+    class OpenapiController < ApplicationController
+      def show
+        render 'show'
```

Comment:
```suggestion
        render :show
```

Reply:
Fixed in https://github.com/nimblehq/nimble-survey-web/pull/89/commits/43a2d65fc23d5253012f31f0b2fcdc505e1b3511